varats/varats/experiments/vara/architecture_report_experiment.py

```diff
@@ -0,0 +1,112 @@
+"""Implements an empty experiment that just compiles the project."""
+
+import typing as tp
+
+from benchbuild import Project
+from benchbuild.extensions import compiler, run, time
+from benchbuild.utils import actions
+from benchbuild.utils.cmd import touch
```

Comment:
You don't use `touch` here, but `opt`.
```suggestion
from benchbuild.utils.cmd import opt
```